packages/components/pie-modal/src/modal.scss

```
@@ -117,6 +117,10 @@
         }
     }
 
+    &[position='top']:not([isfullwidthbelowmid]) {
+        margin-block-start: var(--dt-spacing-j);
+    }
```

Fixes a small bug where the modal positioning stays centred above the breakpoint when `isFullWidthBelowMid` is `true` even though `position` is set to `"top"`.

```suggestion
    &[position='top'] {
        margin-block-start: var(--dt-spacing-j);

        &[isfullwidthbelowmid] {
            @media (max-width: calc($breakpoint-wide - 1px)) {
                margin-block-start: var(--dt-spacing-none);
            }
        }
    }
```